admiral/pkg/controller/admiral/globaltraffic_test.go

```diff
@@ -74,28 +76,217 @@ func TestGlobalTrafficAddUpdateDelete(t *testing.T) {
 
 }
 
-func TestGlobalTrafficGetByLabel(t *testing.T) {
-	config, err := clientcmd.BuildConfigFromFlags("", "../../test/resources/admins@fake-cluster.k8s.local")
-	if err != nil {
-		t.Errorf("%v", err)
+func TestGlobalTrafficController_Updated(t *testing.T) {
+
+	gth := test.MockGlobalTrafficHandler{}
```

Comment:
Nitpick: Can all the variable definition/declaration be added inside
```
var (
)
```
block.

Reply:
Similar comment for `TestGlobalTrafficController_Added` and `TestGlobalTrafficController_Deleted`